std_msgs/Header header

# unit: 0.01 deg
int16 heading

# unit: mm/s
uint16 zupt_rms

# unit: 0.1 deg
uint8 heading_rms
